TaskMarsCompetition/TestMarsCompetition/Tests/TestCertificate.cs

```diff
@@ -0,0 +1,493 @@
+﻿using System;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TestMarsCompetition.Context;
+using TestMarsCompetition.ModelCert;
+
+using TestMarsCompetition.Page;
+
+
+using TestMarsCompetition.Utilities;
+using OpenQA.Selenium.DevTools.V124.Runtime;
+
+
+
+namespace TestMarsCompetition.Tests
+{
+     class TestCertificate:CommonHooks
+    {
+
+        
+        private TestDataCert testDatacert;  
+        private Login login;
+        private CertificatePage certificatePage;
+        private AssertionCert assertions;
+        
+
+        public TestCertificate() {
+
+            
+            
+            certificatePage = new CertificatePage();
+            
+            assertions = new AssertionCert();
+            
+
+
+        }
+
+
+        [Test, Order(1), Description("TC_001 Validate if the creation of Certificate is successful.")]
+
+        public void TC_001_CreateANewCertificationRecord()
+        {
+
+            //Go to Tc01 in the Json input file
+            testDatacert = JsonReaderCert.ReadTestData("Utilities/TestDataCertificate-TC001.json");
+            var testCase = testDatacert.TestCases.Find(tc => tc.TestCaseId == "Tc01");
+            
+          
+            //Get the input data for Certification
+            var certData = testCase.InputData.certificationData;
+       
+            
+            //Add Elements
+            certificatePage.AddCert(certData.certificationName, certData.certificationFrom, certData.certificationYear);
+
+            //Collecting the elemnts added in the particular scenario
+            TestContextManager.AddedCertData.Add(certData.certificationName);
+
+
+            //Assertions to verify addition
+            assertions.AddCertAssert(certData.certificationName, certData.certificationFrom, certData.certificationYear);
+            Thread.Sleep(2000);
+
```

Comment:
ideally here you should clean up the test data that this test adds but for each test you will. need to add that step